### ISSUE TITLE
Running the example, I get an error when ping'ing

### ISSUE THREAD

**Issue by dacarson** [dieselrabbit/screenlogicpy#46]
Hi, I am using the python script to get async updates of my Pentair system. However, when I run it, I occasionally get this error:

```
2023-03-06 11:55:38 DEBUG    keepalive task <Task pending name='Task-4' coro=<ClientManager._async_ping() running at /home/pi/.local/lib/python3.9/site-packages/screenlogicpy/client.py:144>> created
2023-03-06 11:55:38 DEBUG    Requesting ping
2023-03-06 11:55:38 ERROR    Task exception was never retrieved
future: <Task finished name='Task-4' coro=<ClientManager._async_ping() done, defined at /home/pi/.local/lib/python3.9/site-packages/screenlogicpy/client.py:144> exception=TypeError("async_request_ping() missing 1 required positional argument: 'max_retries'")>
Traceback (most recent call last):
  File "/home/pi/.local/lib/python3.9/site-packages/screenlogicpy/client.py", line 151, in _async_ping
    if await async_request_ping(self._protocol):
TypeError: async_request_ping() missing 1 required positional argument: 'max_retries'
```

Is the async callback function missing a parameter, the max_retries parameter?

**Comment by dieselrabbit** [comment]
It sure is. Thanks for the report. I'll get this fixed up asap.

**Comment by dieselrabbit** [comment]
Fixed by 95845f9
